<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainView</class>
 <widget class="QMainWindow" name="MainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainView</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1010</width>
      <height>670</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="Etats">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <italic>false</italic>
      </font>
     </property>
     <attribute name="title">
      <string>Etat station</string>
     </attribute>
     <widget class="QGroupBox" name="Pistes">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>721</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Pistes</string>
      </property>
      <widget class="QRadioButton" name="piste3ferme">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>58</y>
         <width>91</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Fermée</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="piste5ferme">
       <property name="geometry">
        <rect>
         <x>355</x>
         <y>138</y>
         <width>91</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Fermée</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="piste3ouvert">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>30</y>
         <width>91</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Ouverte</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="piste5txt">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>108</y>
         <width>75</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Piste 5 :</string>
       </property>
      </widget>
      <widget class="QLabel" name="piste4txt">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>108</y>
         <width>75</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Piste 4 :</string>
       </property>
      </widget>
      <widget class="QLabel" name="piste3txt">
       <property name="geometry">
        <rect>
         <x>535</x>
         <y>28</y>
         <width>75</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Piste 3 :</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="piste1ouvert">
       <property name="geometry">
        <rect>
         <x>95</x>
         <y>30</y>
         <width>91</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Ouverte</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="piste6ferme">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>138</y>
         <width>91</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Fermée</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="piste2txt">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>28</y>
         <width>75</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Piste 2 :</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="piste5ouvert">
       <property name="geometry">
        <rect>
         <x>355</x>
         <y>110</y>
         <width>91</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Ouverte</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="piste1ferme">
       <property name="geometry">
        <rect>
         <x>95</x>
         <y>58</y>
         <width>91</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Fermée</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="piste2ferme">
       <property name="geometry">
        <rect>
         <x>355</x>
         <y>58</y>
         <width>91</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Fermée</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="piste4ouvert">
       <property name="geometry">
        <rect>
         <x>95</x>
         <y>110</y>
         <width>91</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Ouverte</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="piste6txt">
       <property name="geometry">
        <rect>
         <x>535</x>
         <y>108</y>
         <width>75</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Piste 6 :</string>
       </property>
      </widget>
      <widget class="QLabel" name="piste1txt">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>28</y>
         <width>75</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Piste 1 :</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="piste2ouvert">
       <property name="geometry">
        <rect>
         <x>355</x>
         <y>30</y>
         <width>91</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Ouverte</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="piste4ferme">
       <property name="geometry">
        <rect>
         <x>95</x>
         <y>138</y>
         <width>91</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Fermée</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="piste6ouvert">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>110</y>
         <width>91</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Ouverte</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="Telesieges">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>350</y>
        <width>721</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>Télésieges</string>
      </property>
      <widget class="QLabel" name="teles1txt">
       <property name="geometry">
        <rect>
         <x>133</x>
         <y>50</y>
         <width>131</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Télésiège 1 :</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="teles1ouvert">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>52</y>
         <width>91</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Ouvert</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="teles2ferme">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>80</y>
         <width>91</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Fermé</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="teles2txt">
       <property name="geometry">
        <rect>
         <x>413</x>
         <y>50</y>
         <width>121</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Télésiège 2 :</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="teles2ouvert">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>52</y>
         <width>91</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Ouvert</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="teles1ferme">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>80</y>
         <width>91</width>
         <height>34</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Fermé</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="AnnulerP">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>510</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ConfirmerP">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>510</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirmer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Message">
     <attribute name="title">
      <string>  Message  </string>
     </attribute>
     <widget class="QGroupBox" name="Messages">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>140</y>
        <width>651</width>
        <height>401</height>
       </rect>
      </property>
      <property name="title">
       <string>Messages</string>
      </property>
      <widget class="QLineEdit" name="message10Edit">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>360</y>
         <width>400</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="message9Edit">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>325</y>
         <width>400</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="message8Edit">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>290</y>
         <width>400</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="message2Edit">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>80</y>
         <width>400</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="message1Edit">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>45</y>
         <width>400</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="message7Edit">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>255</y>
         <width>400</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="message3Edit">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>115</y>
         <width>400</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="message4Edit">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>150</y>
         <width>400</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="message5Edit">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>185</y>
         <width>400</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="messagetxt">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Message actuel :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="message6Edit">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>220</y>
         <width>400</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="AnnulerM">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>560</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ConfirmerM">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>560</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirmer</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="titretxt">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>551</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Gestion de la station</string>
    </property>
   </widget>
   <widget class="QLabel" name="tempTxt">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Digital dream</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>° C</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="tempLCD">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>140</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="aideTxt">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>600</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Aide</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="heureTxt">
    <property name="geometry">
     <rect>
      <x>795</x>
      <y>95</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>piste3ferme</tabstop>
  <tabstop>piste5ferme</tabstop>
  <tabstop>piste3ouvert</tabstop>
  <tabstop>piste1ouvert</tabstop>
  <tabstop>piste6ferme</tabstop>
  <tabstop>piste5ouvert</tabstop>
  <tabstop>piste1ferme</tabstop>
  <tabstop>piste2ferme</tabstop>
  <tabstop>piste4ouvert</tabstop>
  <tabstop>piste2ouvert</tabstop>
  <tabstop>piste4ferme</tabstop>
  <tabstop>piste6ouvert</tabstop>
  <tabstop>teles1ouvert</tabstop>
  <tabstop>teles2ferme</tabstop>
  <tabstop>teles2ouvert</tabstop>
  <tabstop>teles1ferme</tabstop>
  <tabstop>AnnulerP</tabstop>
  <tabstop>ConfirmerP</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>message1Edit</tabstop>
  <tabstop>message2Edit</tabstop>
  <tabstop>message3Edit</tabstop>
  <tabstop>message4Edit</tabstop>
  <tabstop>message5Edit</tabstop>
  <tabstop>message6Edit</tabstop>
  <tabstop>message7Edit</tabstop>
  <tabstop>message8Edit</tabstop>
  <tabstop>message9Edit</tabstop>
  <tabstop>message10Edit</tabstop>
  <tabstop>AnnulerM</tabstop>
  <tabstop>ConfirmerM</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
